### PR TITLE
Added support for tabs in asciidoc files

### PR DESCRIPTION
**Pull request by iMashtak**
Resolved https://github.com/apache/camel-website/issues/997

### CODE REVIEW

**Comment by orpiske** [comment]
Wow, if it's going to look like on the issue, IMHO, it will look awesome. Great one @iMashtak!

**Review by oscerd — APPROVED** [review]
Nice @zregvart can you have a look? Thx

File: antora-ui-camel/public/_/partials/head-styles.hbs [hunk 1]
```diff
@@ -1 +1,2 @@
-    <link rel="stylesheet" href="{{uiRootPath}}/css/site-00b7d111da.css">
+    <link rel="stylesheet" href="{{uiRootPath}}/css/site-66db361433.css">
+    <link rel="stylesheet" href="{{{uiRootPath}}}/css/tabs.css">
```
Comment (zregvart) [inline antora-ui-camel/public/_/partials/head-styles.hbs:2]: I don't think this is needed, `tabs.css` is included in the `site-*.css`

File: antora-ui-camel/src/css/site.css [hunk 2]
```diff
@@ -29,3 +29,4 @@
 @import 'catalog.css';
 @import 'sharingbuttons.css';
 @import 'security.css';
+@import 'tabs.css'
```
Comment (zregvart) [inline antora-ui-camel/src/css/site.css:32]: Try this instead, so we don't need to copy `tabs.css` from @asciidoctor/tabs
```suggestion
@import '@asciidoctor/tabs/dist/css/tabs.css'
```

File: package.json [hunk 3]
```diff
@@ -35,6 +35,7 @@
   "devDependencies": {
     "@antora/cli": "^3.0.0",
     "@antora/site-generator": "^3.0.0",
+    "@asciidoctor/tabs": "1.0.0-beta.3",
```
Comment (zregvart) [inline package.json:38]: I don't think we need it here, only in the UI

File: antora-ui-camel/src/partials/head-styles.hbs [hunk 4]
```diff
@@ -1 +1,2 @@
     <link rel="stylesheet" href="{{uiRootPath}}/css/site.css">
+    <link rel="stylesheet" href="{{{uiRootPath}}}/css/tabs.css">
```
Comment (zregvart) [inline antora-ui-camel/src/partials/head-styles.hbs:2]: Should be included in the `site.css`

File: antora-ui-camel/src/css/tabs.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
+/*! Asciidoctor Tabs | Copyright (c) 2018-present Dan Allen | MIT License */
```
Comment (zregvart) [inline antora-ui-camel/src/css/tabs.css:1]: Let's use `@import '@asciidoctor/tabs/...` from above so we don't need to vendor this file, keeping it here makes it prone to being stale when we upgrade the @asciidoctor/tabs dependency

**Review by zregvart — COMMENTED** [review]
Thanks, nice work! Left a few comments for polishing this up

**Review by zregvart — APPROVED** [review]
Thanks!

**Review by oscerd — APPROVED** [review]

**Review by orpiske — APPROVED** [review]

**Comment by davsclaus** [comment]
@zregvart is this okay to merge. @iMashtak and if so, can you maybe do a PR for one of the existing components to show how to do this, and then we can use this as baseline for migrating docs to this style.

**Comment by iMashtak** [comment]
@davsclaus created PR: https://github.com/apache/camel/pull/9992

**Pull request merged**